etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/controller/DataPipelineController.java

```diff
@@ -37,7 +41,7 @@ public ResponseEntity<String> getDataPipelineStatusHealth() {
     public ResponseEntity<String> postProvider(@RequestBody String payLoad) throws JsonProcessingException {
         KafkaProducer<String, JsonNode> producer = new KafkaProducer<>(
                 kafkaStreamsConfig.kStreamsConfigs().asProperties());
-        producer.send(new ProducerRecord<>("cdc.nbs_odse.dbo.Provider",
+        producer.send(new ProducerRecord<>(kafkaConfig.getPersonTopicName(),
```

Comment:
We would need to create this function for each topic, correct?

Reply:
Yeah but This is just for testing purposes. The actual data event should be sent by Debezium Change Data Capture module.